### ISSUE TITLE
Plz separate the form POST URL from the form

### ISSUE THREAD

**Issue by VincentTam** [hendrixjoseph/blog-comments#4883]
# Motivation

To avoid search bots from getting the URL of your API instance (and possibly abusing that): https://www.willmaster.com/library/manage-forms/bot-block-for-forms.php.

https://github.com/hendrixjoseph/puppy-snuggles/blob/6f2efe5d807995a4d31f4795e161b1b6a5ff2439/_includes/comments/form.html#L1-L2

# Implementation

For a detailed implementation, you may view, for example, my PR daattali/beautiful-jekyll<span>#</span>521.

# Other suggestions

I wonder if jQuery in your site can be replaced with jQuery slim, which doesn't include the `ajax` function used in Staticman's author's JavaScript—but that can be replaced by an `xmlHttpRequest`: my PR daattali/beautiful-jekyll<span>#</span>782 shows how.

**Comment by hendrixjoseph** [comment]
Is it that easy? I updated it to my main blog at [commit 041e5fefb08b93951e85302f0fef691678a03fd9](https://github.com/hendrixjoseph/hendrixjoseph.github.io/commit/041e5fefb08b93951e85302f0fef691678a03fd9). This repo gets the comments for that site.

As to the jQuery suggestion, I've completely stripped jQuery out of my other site and just use JavaScript's fetch API. I wrote about that process [here](https://github.com/hendrixjoseph/hendrixjoseph.github.io/commit/041e5fefb08b93951e85302f0fef691678a03fd9). I just need to update the code for the puppy snuggles site.
